app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -399,30 +404,46 @@ const MakeDecision = ({ data }) => {
             modalState={modalState}
             setModalState={setModalState}
             isBulkOperation={isBulkOperation}
-            collapseTabsExceptCurr={collapseTabsExceptCurr}
             collapsedRightSection={collapsedRightSection}
             windowSize={windowSize}
             eventsInfo={data.eventsInfo.editDefectsEvents}
           />
         ),
       },
+      {
+        id: MACHINE_LEARNING_SUGGESTIONS,
+        shouldShow: isMLSuggestionsAvailable,
+        isOpen: tabs[MACHINE_LEARNING_SUGGESTIONS],
+        title: <div>{formatMessage(messages.machineLearningSuggestions)}</div>,
+        content: isMLSuggestionsAvailable && (
+          <MachineLearningSuggestions
+            modalState={modalState}
+            setModalState={setModalState}
+            itemData={itemData}
+            loadingMLSuggest={loadingMLSuggest}
+            eventsInfo={data.eventsInfo.editDefectsEvents}
+            isAnalyzerAvailable={isAnalyzerAvailable}
+          />
+        ),
+      },
     ];
     if (preparedHistoryLineItems.length > 0) {
-      tabsData.splice(1, 0, {
+      tabsData.splice(2, 0, {
         id: COPY_FROM_HISTORY_LINE,
         shouldShow: !isBulkOperation,
-        disabled: false,
         isOpen: tabs[COPY_FROM_HISTORY_LINE],
-        title: formatMessage(messages.copyFromHistoryLine),
+        title: formatMessage(messages.executionWith, {
+          value: '100',
```

Comment:
This section contains items from history line. Are you sure that this message should be shown here?
Better to name it as on previous design `Copy from history line`.